zarr/tests/test_core.py

```diff
@@ -1631,6 +1627,10 @@ def test_astype(self):
         expected = data.astype(astype)
         assert_array_equal(expected, z2)
 
+    def test_unstructured_array(self):
+        # skip this one, cannot do delta on unstructured array
```

Comment:
Technicality: numcodecs cannot do deltas on unstructured array dtypes
in string form e.g. `"(4, 4)f4"`. Since the expanded dtype is already
tested in `test_dtypes`, skip instead.